### PR TITLE
operators-validators-owner-limit-IO1-728

### PR DESCRIPTION
**Pull request by vadiminc**

### CODE REVIEW

File: test/unit/ssv-registry.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ describe('SSV Registry', function () {
 
   it('Owner address limit', async function () {
     expect((await ssvRegistry.getOperatorsByOwnerAddress(account2.address)).length).to.equal(2);
-    expect(await ssvRegistry.getOperatorsPerOwnerLimit()).to.equal(200);
+    expect(await ssvRegistry.getOperatorsPerOwnerLimit()).to.equal(10);
```
Comment (andrew-blox) [inline test/unit/ssv-registry.ts:58]: change name to getRegisteredOperatorsPerAccountLimit

File: test/unit/update-operators.ts [hunk 2]
```diff
@@ -39,7 +40,7 @@ describe('Update Operators', function () {
     ssvRegistry = await upgrades.deployProxy(ssvRegistryFactory, { initializer: false })
     await ssvToken.deployed()
     await ssvRegistry.deployed()
-    ssvNetwork = await upgrades.deployProxy(ssvNetworkFactory, [ssvRegistry.address, ssvToken.address, minimumBlocksBeforeLiquidation, operatorMaxFeeIncrease, setOperatorFeePeriod, approveOperatorFeePeriod, validatorsPerOperatorLimit, operatorsPerOwnerLimit])
+    ssvNetwork = await upgrades.deployProxy(ssvNetworkFactory, [ssvRegistry.address, ssvToken.address, minimumBlocksBeforeLiquidation, operatorMaxFeeIncrease, setOperatorFeePeriod, approveOperatorFeePeriod, validatorsPerOperatorLimit, operatorsPerOwnerLimit, operatorsPerValidatorsOwnerLimit])
```
Comment (andrew-blox) [inline test/unit/update-operators.ts:43]: update names
registeredOperatorsPerAccountLimit
managingOperatorsPerAccountLimit

File: test/unit/update-validators.ts [hunk 3]
```diff
@@ -39,7 +40,7 @@ describe('Update Validators', function () {
     ssvRegistry = await upgrades.deployProxy(ssvRegistryFactory, { initializer: false })
     await ssvToken.deployed()
     await ssvRegistry.deployed()
-    ssvNetwork = await upgrades.deployProxy(ssvNetworkFactory, [ssvRegistry.address, ssvToken.address, minimumBlocksBeforeLiquidation, operatorMaxFeeIncrease, setOperatorFeePeriod, approveOperatorFeePeriod, validatorsPerOperatorLimit, operatorsPerOwnerLimit])
+    ssvNetwork = await upgrades.deployProxy(ssvNetworkFactory, [ssvRegistry.address, ssvToken.address, minimumBlocksBeforeLiquidation, operatorMaxFeeIncrease, setOperatorFeePeriod, approveOperatorFeePeriod, validatorsPerOperatorLimit, operatorsPerOwnerLimit, operatorsPerValidatorsOwnerLimit])
```
Comment (andrew-blox) [inline test/unit/update-validators.ts:43]: update names
registeredOperatorsPerAccountLimit
managingOperatorsPerAccountLimit

File: test/unit/validators.ts [hunk 4]
```diff
@@ -38,7 +39,7 @@ describe('Validators', function () {
     ssvRegistry = await upgrades.deployProxy(ssvRegistryFactory, { initializer: false })
     await ssvToken.deployed()
     await ssvRegistry.deployed()
-    ssvNetwork = await upgrades.deployProxy(ssvNetworkFactory, [ssvRegistry.address, ssvToken.address, minimumBlocksBeforeLiquidation, operatorMaxFeeIncrease, setOperatorFeePeriod, approveOperatorFeePeriod, validatorsPerOperatorLimit, operatorsPerOwnerLimit])
+    ssvNetwork = await upgrades.deployProxy(ssvNetworkFactory, [ssvRegistry.address, ssvToken.address, minimumBlocksBeforeLiquidation, operatorMaxFeeIncrease, setOperatorFeePeriod, approveOperatorFeePeriod, validatorsPerOperatorLimit, operatorsPerOwnerLimit, operatorsPerValidatorsOwnerLimit])
```
Comment (andrew-blox) [inline test/unit/validators.ts:42]: update names
registeredOperatorsPerAccountLimit
managingOperatorsPerAccountLimit

**Review by andrew-blox — CHANGES_REQUESTED** [review]

File: .env.example [hunk 5]
```diff
@@ -10,4 +10,6 @@ MINIMUM_BLOCKS_BEFORE_LIQUIDATION=100
 OPERATOR_MAX_FEE_INCREASE=3
 SET_OPERATOR_FEE_PERIOD=259200 # 3 days
 APPROVE_OPERATOR_FEE_PERIOD=345600 # 4 days
-VALIDATORS_PER_OPERATOR_LIMIT=2000
\ No newline at end of file
+VALIDATORS_PER_OPERATOR_LIMIT=2000
+OPERATORS_PER_OWNER_LIMIT=10
+OPERATORS_PER_VALIDATORS_OWNER_LIMIT=50
```
[the remark this answers is not in the document]
Comment (vadiminc) [reply .env.example:15]: what this comment means? I used as described in PRD

**Pull request merged**